activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Adjust minimum MariaDB version for `CHECK_CONSTRAINTS` table.
```

Comment:
How about "Adjust the minimum MariaDB version for check constraints support." ? `CHECK_CONSTRAINTS` is something internal one, not for Rails application developers.

Reply:
Updated.